docs/decisions/0001-add-runtime-resources-migration-configuration.md

```diff
@@ -0,0 +1,89 @@
+---
+# These are optional metadata elements. Feel free to remove any of them.
+status: "proposed"
+date: 2025-10-02
+---
+
+# Camunda 7 to Operaton Runtime Resources Migration
+
+## Context and Problem Statement
+
+There is a problem with code migration for production runtimes if any platform code with FQN was used in resources like scripts or configurations. It is necessary to migrate it in a database
+
+<!-- This is an optional element. Feel free to remove. -->
+## Decision Drivers
```

Comment:
Please format decision drivers as a bullet list for clarity. Example:

- Must enable smooth migration from Camunda to Operaton without data loss  
- Must remain DB-agnostic  
- Must allow parallel cluster operation (Camunda + Operaton)  
- Should avoid manual SQL scripts  
- Should not break existing deployments